lahja/base.py

```diff
@@ -206,11 +225,9 @@ def broadcast_nowait(
     # Subscription API
     #
     @abstractmethod
-    def get_connected_endpoints_and_subscriptions(
-        self
-    ) -> Tuple[Tuple[Optional[str], Set[Type[BaseEvent]]], ...]:
+    async def wait_until_remote_subscriptions_change(self) -> None:
         """
-        Return all connected endpoints and their event type subscriptions to this endpoint.
+        Block until any subscription change occurs on any remote endpoint.
```

Comment:
nitpick: Maybe this comment could also note that this method will also return when the set of remote endpoints changes.